test/requests/on_type_formatting_test.rb

```diff
@@ -292,6 +292,32 @@ def test_comment_continuation
     assert_equal(expected_edits.to_json, T.must(edits).to_json)
   end
 
+  def test_comment_continuation_does_not_apply_to_rbs_signatures
+    document = RubyLsp::RubyDocument.new(
+      source: +"",
+      version: 1,
+      uri: URI("file:///fake.rb"),
+      global_state: @global_state,
+    )
+
+    document.push_edits(
+      [{
+        range: { start: { line: 0, character: 0 }, end: { line: 0, character: 0 } },
+        text: "    #: (String) -> String",
```

Comment:
Let's maybe also add an example with a trailing `#:` comments to ensure we don't accidentally change the regexp?

This should not trigger:
```rb
a = 42 #: Integer
```

Reply:
Added